### PR TITLE
[feat]:grids

### PR DESCRIPTION
**Pull request by ultronjs**
Added Grid With:-
2 Items in a column
3 Items in a column

### CODE REVIEW

File: css/components/grid.css [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+.col-2-shrink{
+    grid-template-columns: 1fr 2fr;
+}
+
+.col-2-expand{
+    grid-template-columns: 2fr 1fr;
+}
+.col-3{
+    grid-template-columns: 1fr 1fr 1fr;
+}
```
Comment (JantuDeb) [inline css/components/grid.css]: you can use `repeat(3, 1fr)` here

**Review by JantuDeb — APPROVED** [review]
This is looking  good

File: css/components/grid.css (hunk 1 above)
Comment (ultronjs) [reply css/components/grid.css]: Done

**Review by Namrata-J — APPROVED** [review]
consistent code, good that you have used px for units lesser than 1rem

**Comment by ultronjs** [comment]
Merging after 2 approval

**Pull request merged**